contracts/lines/base/BaseMessageLine.sol

```diff
@@ -3,10 +3,14 @@
 pragma solidity ^0.8.0;
 
 import "../../interfaces/IMessageLine.sol";
-import "./LineMetadata.sol";
+import "../../interfaces/ILineMetadata.sol";
```

Comment:
Can we remove `ILineMetadata` and then move the `name()` interface to `IMessageLine.sol` ?

Reply:
Add json metadata